pkg/storage/bench_test.go

```diff
@@ -1681,6 +1748,37 @@ func runMVCCExportToSST(b *testing.B, numKeys int, numRevisions int, exportAllRe
 
 		require.Equal(b, assertLen, buf.Len())
 	}
+
+	// Run sanity checks on last produced SST.
+	it, err := NewPebbleMemSSTIterator(
+		buf.Bytes(), true /* verify */, IterOptions{
+			LowerBound: keys.LocalMax,
+			UpperBound: roachpb.KeyMax,
+			KeyTypes:   IterKeyTypePointsAndRanges,
+		},
+	)
+	it.SeekGE(MakeMVCCMetadataKey(roachpb.LocalMax))
+	require.NoError(b, err)
+	var n int // points
+	var r int // range key stacks
```

Comment:
nit: This can be off by as much as 2x, because it's incremented both when we step onto and off of a stack. Still works for our purposes here though.